front/lib/swr.ts

```diff
@@ -320,7 +321,7 @@ export function useDataSources(
   };
 }
 
-export function useMembers(owner: WorkspaceType) {
+export function useMembers(owner: WorkspaceType | LightWorkspaceType) {
```

Comment:
Type inheritance
```suggestion
export function useMembers(owner: LightWorkspaceType) {
```

Reply:
`useMembers` is used in the workspace members view and owner is of type `WorkspaceType` there

Reply:
Yep but IIRC, `WorkspaceType` extends `LightWorkspaceType` so you should be good to go with `LightWorkspaceType`